packs/feats/earned-glory.json

```diff
@@ -42,7 +42,7 @@
                     "target:trait:elf"
                 ],
                 "selector": "performance",
-                "text": "PF2E.SpecificRule.HalfElf.EarnedGlory.Note",
+                "text": "PF2E.SpecificRule.Aiuvarin.EarnedGlory.Note",
```

Comment:
This is going to break localization for all instances of this feat out in the wild